### DISCUSSION TITLE
Is this nom8?

### DISCUSSION THREAD

**Discussion by flynnblaa** [winnow-rs/winnow#94] [Q&A] [answered]
I'm here because I followed the link on the crates.io page for nom8 - which points to this repo. At first I thought it was a wrong link, but on closer inspection, I think winnow is a fork from nom that was briefly (pre-development) known as nom8. Perhaps nom8 was never intended to be seen by the public? 

I was interested in nom8 because it's a dependency of toml-edit. If I'm correct with the above, are there any plans to update the dependency of toml-edit to be winnow?

**Comment by epage** [comment, marked as answer]
`nom8` was a short-lived fork of `nom` to prototype ideas for `nom` v8's development.  As I felt these were blockers for using `nom` in `toml_edit`, I went ahead and published it rather than wait on `nom` v8.   As I've further explored the needs of `toml_edit` and discussed them with nom's maintainer, I felt it was best to make a longer-term fork for exploring these ideas, hence `winnow`.

`toml_edit` will be updated when the first release of `winnow` is made and I will write up a blog post with more details on "why `winnow`" and the long term goals.

**Reply by epage** [reply]
FYI `winnow` is out: https://epage.github.io/blog/2023/02/winnow-toml-edit-combine-nom/
